sites/docs/src/content/docs/usage/Getting_started/configuration.md

```diff
@@ -138,7 +162,38 @@ Where possible we try to get tools to make use of the resources available, for e
 
 To tune workflow resources to better match your requirements, we can tweak these through [custom configuration files](#custom-configuration-files) or [shared nf-core/configs](#shared-nf-coreconfigs).
 
-By default, most process resources are specified using process _labels_, for example with the following base config:
+By default, most process resources are specified using process _labels_, as in the following example base config:
+
+```groovy
+process {
+  resourceLimits = [
+    cpus: 32,
+    memory: 256.GB,
+    time: 24.h
+  ]
+  withLabel:process_low {
+    cpus   = { 2 * task.attempt }
+    memory = { 14.GB * task.attempt }
+    time   = { 6.h  * task.attempt }
+  }
+  withLabel:process_medium {
+    cpus   = { 6  * task.attempt }
+    memory = { 42.GB * task.attempt }
+    time   = { 8.h * task.attempt }
+  }
+  withLabel:process_high {
+    cpus   = { 12 * task.attempt }
+    memory = { 84.GB * task.attempt }
+    time   = { 10.h * task.attempt }
+  }
+}
+```
+
+The `resourceLimits` list sets absolute maximums any pipeline job can request (typically corresponding to the maximum available resources on your machine). The label blocks indicate the initial 'default' resources a pipeline job will request. For most nf-core pipelines, if a job runs out of memory, it will re-try the job but increasing the amount resource requested up to the `resourceLimits` maximum.
+
+:::note{collapse title="Note on older nf-core pipelines"}
+
+If you wish to use pipelines generated with the pre-`v3.0.0` nf-core template, and/or when running with Nextflow versions earlier than 24.04.0 you may need use a different syntax to prevent the resources from exceeding a maximum limit.
```

Comment:
```suggestion
If you wish to use pipelines generated with the nf-core template before `v3.0.0`, and/or when running with Nextflow versions earlier than 24.04.0 you may need use a different syntax to prevent the resources from exceeding a maximum limit.
```